src/main/java/net/modfest/scatteredshards/item/ShardTablet.java

```diff
@@ -0,0 +1,36 @@
+package net.modfest.scatteredshards.item;
+
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.sound.PositionedSoundInstance;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.sound.SoundEvents;
+import net.minecraft.util.Hand;
+import net.minecraft.util.TypedActionResult;
+import net.minecraft.world.World;
+import net.modfest.scatteredshards.client.screen.ShardTabletGuiDescription;
+import net.modfest.scatteredshards.component.ScatteredShardsComponents;
+
+public class ShardTablet extends Item {
+
+    public ShardTablet(Settings settings) {
+        super(settings);
+    }
+
+    // TODO: sneak interact on another player opens their collection
+    @Override
+    public TypedActionResult<ItemStack> use(World world, PlayerEntity user, Hand hand) {
+        if (!world.isClient) {
+            return TypedActionResult.pass(user.getStackInHand(hand));
+        }
+        final var library = ScatteredShardsComponents.getShardLibrary(world);
+        final var collection = ScatteredShardsComponents.getShardCollection(user);
+        final var client = MinecraftClient.getInstance();
+        client.execute(() -> {
```

Comment:
```java
client.send(() -> { });
```
to get off the tick and yeet the item-use from the stack trace so LibGui will never know.

Don't forget to fix this file to tabs instead of spaces.